crates/typst-layout/src/math/frac.rs

```diff
@@ -20,14 +22,28 @@ pub fn layout_frac(
     ctx: &mut MathContext,
     styles: StyleChain,
 ) -> SourceResult<()> {
-    layout_frac_like(
-        ctx,
-        styles,
-        &elem.num,
-        std::slice::from_ref(&elem.denom),
-        false,
-        elem.span(),
-    )
+    match elem.style.get(styles) {
+        FracStyle::Skewed => {
+            layout_skewed_frac(ctx, styles, &elem.num, &elem.denom, elem.span())
+        }
+        FracStyle::Horizontal => layout_horizontal_frac(
+            ctx,
+            styles,
+            &elem.num,
+            &elem.denom,
+            elem.span(),
+            elem.num_deparenthesized.as_option().unwrap_or(false),
```

Comment:
```suggestion
            elem.num_deparenthesized.get(styles),
```
This'll make use of the default set in the FracElem definition. Similar for denom below.